content/docs/operating/security.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Security
+sort_rank: 4
+---
+
+# Security Model
+
+Prometheus is sophisticated system with many components and many integrations with
+other systems. It can be deployed in a varietry of trusted and untrusted
+enviroments.
+
+This page describes the general security assumptions of Prometheus and the
+attack vectors that some configurations may enable.
+
+As with any complex systems it is not possible to gaurantee that there are no
+bugs. If you find a security bug, please file it in the issue tracker of the
+relevant component.
+
+### Prometheus
+
+It is presumed that untrusted users have access to the Prometheus HTTP endpoint
+and logs. They have access to all time series information contained in the
+database, plus a variety of operational/debugging information.
```

Comment:
I know, but it's 1-2 more words and as it's written right now, it's not 100% correct.